### PR TITLE
Add qs options passthrough

### PR DESCRIPTION
**Pull request by akshaykarthik**
I'm using restify in my application and I'm passing through options to Apache Solr.
The way Solr parses query strings are different than the way `qs` does by default.

`qs` supports many options to modify the parsing behavior and a passthrough would be useful.

### CODE REVIEW

**Comment by DonutEspresso** [comment]
This LGTM. Can I get a second set of eyes?

File: lib/plugins/query.js [hunk 1]
```diff
@@ -14,12 +14,16 @@ var assert = require('assert-plus');
...
  * @function queryParser
  * @param    {Object}   options an options object
+ * @param    {Object}   qsOptions an options object for qs
  * @returns  {Function}
  */
-function queryParser(options) {
+function queryParser(options, qsOptions) {
 
     var opts = options || {};
+    var qsOpts = qsOptions || {};
```
Comment (micahr) [inline lib/plugins/query.js:23]: It doesn't look like `qsOpts` is used in `parseQuerystring`. Is that intentional?

Comment (yunong) [reply lib/plugins/query.js:23]: @akshaykarthik ping

Comment (akshaykarthik) [reply lib/plugins/query.js:23]: Sorry, this is an oversight, will fix it now.

**Comment by akshaykarthik** [comment]
This is no longer necessary, #39 solves this issue

**Pull request closed without merging**